rules/0.3-allow-points-transfer.py

```diff
@@ -18,6 +18,9 @@ def should_allow(pr):
     #  - me:  -1 point
     #  - you: +1 point
 
+    if not pr.creates_bonus():
+        return True
```

Comment:
Returning true here means that any PR that isn't creating bonuses is allowed to merge without checking any other rules. Is that what you intended?